### PR TITLE
Apple website webpage feature a

### PR DESCRIPTION
**Pull request by Elbie-em**
-Added Nav element with gradient styling
-Added Main image and centered text
-Added Thumbnails using grid
-Added footer with floats

### CODE REVIEW

**Review by nikasulo — CHANGES_REQUESTED** [review]
# Under Review By Tolulope Olusakin(TSE)
I am currently writing your review! Please check on this within the hour for some feedback :smile:

**Comment by Elbie-em** [comment]
Will do so, thank you.

Kind regards

Elbie Moonga

**Review by nikasulo — CHANGES_REQUESTED** [review]
# Nice work @Elbie-em! :rocket: 
Please look into my comments below, address the issues and request another review :100: 

#### Horizontal Scroll
You seem to have a little bug with one of the elements of the page, shooting out beyond the browser's maximum width. This may be as a result of a margin or padding you have set up. One of the easiest ways I have found to correct this issue is to set the default `box-sizing` property to `border-box` as follows `*{box-sizing: border-box}`. Please read some more about it [here](https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing).
screenshot

#### Too much spacing
Please consider looking into the spacing between your navigation bar and the text beneath it, compare it with the original site, and make appropriate changes! I for one think it could use a reduction in size and I have attached a screenshot for your viewing.
screenshot

### Summary
You have done a really nice job so far, please look into the few changes I am proposing and request a re-review! :handshake: See you at the top :medal_military: 

##### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+  font-size: 36px;
+}
+
+.main_content p:hover {
+  cursor: pointer;
+}
+
+.main_content_b {
+  display: grid;
+  grid-template-columns: 25vw 25vw 25vw 25vw;
```
Comment (Forison) [inline styles.css]: * [x] change ```grid-template-columns: 25vw 25vw 25vw 25vw;``` to ```grid-template-columns: 25% 25% 25% 25%;```

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+  height: auto;
+  margin: 2px;
+}
+
+.thumbnail {
+  height: auto;
+  width: 25vw;
```
Comment (Forison) [inline styles.css]: * [x]  Change ```width: 25vw;``` to ``` width: 100%;```

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+  cursor: pointer;
+}
+
+footer {
+  width: 100vw;
```
Comment (Forison) [inline styles.css]: * [x] Change ```width: 100vw;``` to ```width: 100%;```
That is it with this the horizontal scroll should not show again

**Review by Forison — CHANGES_REQUESTED** [review]
### Status: Require changes🛑
# Hi @Elbie-em🙋‍♂️ 
You have done a good job on this one, here are a few changes we ave to do to get this project approved🙂
* [x] Files not related to the project should not be committed and added to .gitignore(e.g. `.DS_Store`) 
([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/gitignore.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)) 
#### Check below for how to resolve the issue👇,

> It is better to set your widths with %, that way they become responsive to different desktop views, this way horizontal scrolls won't be seen on desktop views lower than yours🙂
## Kindly make these changes and submit the project for review🙂
Addo Forison

**Review by mosesogwo — CHANGES_REQUESTED** [review]
**Hi @Elbie-em, :clap:**
1. The horizontal scroll is still not fixed because your `.main_content` is still sized in **vw**. Use **%** as you have been previously advised.

2. While taking care of that, reduce the font-size of the text in the section below and add some margins to the left and right of the text. Yours occupy almost 100% of the page width, a sharp contrast from the original. Cross check with the original page and make it similar.
image

_**Kindly make these changes and submit another code review request.**_

**Best :smile:**

**Review by mosesogwo — APPROVED** [review]
**:trophy:Congratulations! You can now :rocket:**.

You may now merge to master.

Note: You are encouraged to state the actions you performed as commit messages instead of generic statements.

**Good luck in your future projects :smile:**

**Comment by Elbie-em** [comment]
-Changed vw to %
-adjusted main section text size
-adjusted margins in main section

**Pull request merged**